Makefile

```diff
@@ -220,7 +139,7 @@ test-with-docker: prepare-docker-build-image create-docker-test-db ## Run tests
 		${DOCKER_BUILDER_IMAGE_NAME} \
 		gosu hostuser make test
 
-.PHONY: test-with-docker
+.PHONY: create-docker-test-db
```

Comment:
🤔

Reply:
Probably a copy/paste leftover. `.PHONY` is used to override the default behaviour of `make` which is to match targets to files.

Reply:
would it have been running both commands when we call test-with-docker?

Reply:
No, it would just look for a file called `create-docker-test-db` when we called `make create-docker-test-db`, before executing the commands.